<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>SolaimanLipi</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Accounts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbViewAccountsRefresh">
       <property name="text">
        <string>Refresh Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAddAccount">
       <property name="text">
        <string>Add New Account</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <family>SolaimanLipi</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>NAME</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ADDRESS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MOBILE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Dr/Cr</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
